ha-singleton-service/README.md

```diff
@@ -3,44 +3,47 @@
 Author: Radoslav Husar  
 Level: Advanced  
 Technologies: MSC, Singleton Service, Clustering  
-Summary: The `ha-singleton-service` quickstart demonstrates deploying a cluster-wide singleton MSC service.
+Summary: The `ha-singleton-service` quickstart demonstrates how to deploy a cluster-wide singleton MSC service.  
 Target Product: ${product.name}  
 Source: <${github.repo.url}>  
 
 
 ## What is it?
 
-The `ha-singleton-services` quickstart demonstrates two patterns of deploying a cluster-wide singleton MSC service.
-The first pattern in the `primary-only` directory demonstrates a singleton service and a querying service which regularly queries the value the singleton service provides.
-It is transparent whether the singleton service is running on the same node or the value is obtained remotely.
-The second pattern in the `with-backups` directory demonstrates a singleton service which is installed with a backup service.
-The backup service is running on all nodes which are not elected to be running the singleton service. 
+The `ha-singleton-services` quickstart demonstrates two patterns, or ways, to deploy a cluster-wide singleton MSC service.
 
-The examples are built and packaged as JAR archives.
+1. The first example, which is located in the `primary-only/` directory of the quickstart, demonstrates a singleton service and a querying service that regularly queries for the value that the singleton service provides. It is transparent whether the singleton service is running on the same node or whether the value is obtained remotely.
```

Comment:
The documentation team always adds a '/' at the end of a directory name to indicate that it is a folder and not a file. I can remove it if you object since this is not the official documentation. Let me know.